lib/Interface/c_interface.cpp

```diff
@@ -2094,3 +2094,96 @@ int vc_parseMemExpr(VC vc, const char* s, Expr* oquery, Expr* oasserts)
   }
   return 1;
 }
+
+void _vc_setSolver(VC vc, stp::UserDefinedFlags::SATSolvers solver)
+{
+  /* Helper method to encapsulate setting a solver */
+  stp::STP* stp_i = (stp::STP*)vc;
+  stp::STPMgr* b = stp_i->bm;
+  b->UserFlags.solver_to_use = solver;
+}
+
+unsigned char _vc_usingSolver(VC vc, stp::UserDefinedFlags::SATSolvers solver)
+{
+  /* Helper method to encapsulate getting a solver */
+  stp::STP* stp_i = (stp::STP*)vc;
+  stp::STPMgr* b = stp_i->bm;
+  return b->UserFlags.solver_to_use == solver;
+}
+
+unsigned char vc_setMinisat(VC vc)
+{
+  _vc_setSolver(vc, stp::UserDefinedFlags::MINISAT_SOLVER);
+  unsigned char success = 1;
```

Comment:
Why not just `return 1`, or better yet, not return any value?

Reply:
I was trying to be explicit and avoid using random ints in the code, but there's no reason I couldn't do that.

I added that "success" option because I wanted the other solvers to return the success too -- for the other solvers, it is possible those methods would not succeed because the build of STP wasn't compiled with support for that solver.

For Minisat, it is 100% fine for this _not_ to return a value because you can't build STP without it.

Reply:
Well, I'd like them all to be the same function prototype, so they can all return a value, but it doesn't look like you're first testing if it's supported before setting it. 

You could use `<stdbool.h>` and return `true`?

Reply:
> Well, I'd like them all to be the same function prototype, so they can all return a value, but it doesn't look like you're first testing if it's supported before setting it.

STP doesn't have a notion of "not using" minisat or simplifying minisat, so I don't think it really makes sense to add that (but I can).

For the optional solvers, the `#ifdef`s should guard against trying to set a solver if it hasn't been compiled in.

Reply:
The change to use `bool` is done.